summoner-cli/test/Test/TomlSpec.hs

```diff
@@ -18,6 +22,16 @@ import qualified Hedgehog.Gen as Gen
 import qualified Hedgehog.Range as Range
 
 
+tomlSpec :: Spec
+tomlSpec = describe "TOML configuration spec" $ do
+    it "finalises default configuration" $
+        finalise defaultConfig `shouldSatisfy` isSuccess
+    it "parses default configuration" $
+        decode configCodec defaultConfigFileContent `shouldSatisfy` isRight
```

Comment:
This is the best I could come up in terms of testing. Ideally, it would be nice to parse with uncommented fields, but I can't just drop `#` in front of each line since some text is the help text.

Reply:
I see.
I think golden testing I described above could help with the issue :slightly_smiling_face: